Extensions/ValuesOfMultipleObjects.json

```diff
@@ -0,0 +1,1472 @@
+  "extensionNamespace": "",
+  "fullName": "Values of multiple objects",
+  "helpPath": "",
+  "iconUrl": "data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz48IURPQ1RZUEUgc3ZnIFBVQkxJQyAiLS8vVzNDLy9EVEQgU1ZHIDEuMS8vRU4iICJodHRwOi8vd3d3LnczLm9yZy9HcmFwaGljcy9TVkcvMS4xL0RURC9zdmcxMS5kdGQiPjxzdmcgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgdmVyc2lvbj0iMS4xIiBpZD0ibWRpLWRvdHMtaGV4YWdvbiIgd2lkdGg9IjI0IiBoZWlnaHQ9IjI0IiB2aWV3Qm94PSIwIDAgMjQgMjQiPjxwYXRoIGQ9Ik0xNiAxMkMxNiAxMC45IDE2LjkgMTAgMTggMTBTMjAgMTAuOSAyMCAxMiAxOS4xIDE0IDE4IDE0IDE2IDEzLjEgMTYgMTJNMTAgMTJDMTAgMTAuOSAxMC45IDEwIDEyIDEwUzE0IDEwLjkgMTQgMTIgMTMuMSAxNCAxMiAxNCAxMCAxMy4xIDEwIDEyTTQgMTJDNCAxMC45IDQuOSAxMCA2IDEwUzggMTAuOSA4IDEyIDcuMSAxNCA2IDE0IDQgMTMuMSA0IDEyTTEzIDE4QzEzIDE2LjkgMTMuOSAxNiAxNSAxNlMxNyAxNi45IDE3IDE4IDE2LjEgMjAgMTUgMjAgMTMgMTkuMSAxMyAxOE03IDE4QzcgMTYuOSA3LjkgMTYgOSAxNlMxMSAxNi45IDExIDE4IDEwLjEgMjAgOSAyMCA3IDE5LjEgNyAxOE0xMyA2QzEzIDQuOSAxMy45IDQgMTUgNFMxNyA0LjkgMTcgNiAxNi4xIDggMTUgOCAxMyA3LjEgMTMgNk03IDZDNyA0LjkgNy45IDQgOSA0UzExIDQuOSAxMSA2IDEwLjEgOCA5IDggNyA3LjEgNyA2IiAvPjwvc3ZnPg==",
+  "name": "ValuesOfMultipleObjects",
+  "previewIconUrl": "https://resources.gdevelop-app.com/assets/Icons/dots-hexagon.svg",
+  "shortDescription": "Values of picked object instances (including position, force, and angle)",
+  "version": "1.2.1",
+  "tags": [
+    "objects",
+    "expression",
+    "calculate",
+    "center",
+    "middle",
+    "average",
+    "position",
+    "flocking",
+    "camera",
+    "mean",
+    "median",
+    "max",
+    "min",
+    "velocity"
+  ],
+  "authorIds": [
+    "gqDaZjCfevOOxBYkK6zlhtZnXCg1",
+    "2OwwM8ToR9dx9RJ2sAKTcrLmCB92"
+  ],
+  "dependencies": [],
+  "eventsFunctions": [
+    {
+      "description": "Minimum X position of picked object instances (using AABB of objects)",
+      "fullName": "Minimum X position of picked object instances",
+      "functionType": "Expression",
+      "name": "MinX",
+      "private": false,
+      "sentence": "",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": "\n// If the instances doesn't exist we exit this function, returnValue will be 0\nif (objects.length == 0) return;\n\nlet minX = objects[0].getAABB().min[0] || 0;\n\nobjects.forEach((object) => {\n\n    let aabb = object.getAABB();\n    \n    if (aabb.min[0] < minX) {\n        minX = aabb.min[0];\n    }\n})\n\neventsFunctionContext.returnValue = minX;",
```

Comment:
> `let minX = objects[0].getAABB().min[0] || 0`

@Bouh In what case exactly would `min[0]` return a falsy value that is not already 0?

Reply:
Also, note that what has been done here is basically an `Array.reduce`, I do not think it is a big deal in that case but just saying in case you didn't know about that function 👀 
```ts
if (objects.length == 0) return;
eventsFunctionContext.returnValue = objects
    .map(object => object.getAABB().min[0])
    .reduce((previous, current) => previous > current ? current : previous)

Reply:
I didn't know about the reduce concept. Thanks for the discovery.

Reply:
> > let minX = objects[0].getAABB().min[0] || 0
> @Bouh In what case exactly would min[0] return a falsy value that is not already 0?
@arthuro555 

I saw that for a non-existing value, in doubt, I used a fallback.
![image](https://user-images.githubusercontent.com/1670670/143001294-77c37c5c-829e-4b69-86e6-560d9d432e15.png)

Reply:
Of course `min[6]` will return undefined as there are always 2 return values not 7 but `min[0]` should never be nullish